pagopa/io-app
# feat: [IOBP-348] Addition of bottomSheet in idPay discount initiative CTA

## Short description

DEPENDS ON https://github.com/pagopa/io-app/pull/5212

addition of bottomSheet in idPay discount type initiatives in sight of the future inclusion of barcode generation
<img src="https://github.com/pagopa/io-app/assets/60693085/c226dfc8-b294-42e4-a686-a5a681a0512a" width=200/>

## List of changes proposed in this pull request
- addition of hook to handle the bottomSheet
- removal of qrcode icon from CTA as of design
- addition of required I18n

## How to test
navigate to an idpay discount initiative, press "authorize payment" and make sure that:
- a bottomSheet appears
- "scan QR" navigates to the current qrcode scan screen
- "create barcode" closes the bottomSheet
- a11y keys are correctly read.
